generation_config.yaml

```diff
@@ -0,0 +1,42 @@
+gapic_generator_version: 2.41.0
+protoc_version: '25.3'
```

Comment:
I don't think we need `protoc_version` and `template_excludes` anymore.

Reply:
Done for all repos.